addons/versioning/XEH_postInitServer.sqf

```diff
@@ -20,6 +20,9 @@ FUNC(paranoid) = {
 
 QGVAR(versions_serv) addPublicVariableEventHandler { (_this select 1) call FUNC(paranoid) };
 
+// Skip missing mod check if it is disabled.
+if (isNumber (configFile >> "CBA_skipMissingModCheck") && {getNumber (configFile >> "CBA_skipMissingModCheck") == 1}) exitWith {};
```

Comment:
Why the `isNumber` check?

Reply:
Not sure if the getNumber could cause errors. But as I am writing this right now I remember the talk from a couple days ago. Fix incoming